code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm

```diff
@@ -152,19 +152,26 @@
 		/datum/design/autolathe/ammo/magnum_rubber,
 		/datum/design/autolathe/ammo/magnum_practice = 0,
 		/datum/design/autolathe/ammo/magnum_lethal,
+		/datum/design/autolathe/ammo/slkurtz,
+		/datum/design/autolathe/ammo/slkurtz_lethal,
```

Comment:
I dont think this is a good idea + this needs to be mirrored to marshals disk to be 1:1

Reply:
![image](https://user-images.githubusercontent.com/65790871/135788267-faf9ea4d-c243-4214-b5ec-382dde10043f.png)
![image](https://user-images.githubusercontent.com/65790871/135788310-45728dfb-ec9d-4dd7-9f22-668853402adc.png)
![image](https://user-images.githubusercontent.com/65790871/135788360-84ab1bb9-0e8d-4588-bb8c-cb344e0bb558.png)
![image](https://user-images.githubusercontent.com/65790871/135788483-e0a94d30-b2a6-4e7d-8dc8-a5d6ffaebc6a.png)
![image](https://user-images.githubusercontent.com/65790871/135788560-4b3ac572-3cd6-48ef-ae9f-d1e04e7e2714.png)
![image](https://user-images.githubusercontent.com/65790871/135788604-433dfd60-31a4-4129-87b0-2b5742ff9370.png)
![image](https://user-images.githubusercontent.com/65790871/135788623-c80bf32d-4a3d-4d2f-b039-2a1acc2b4400.png)
Hope the pictures work, if not, I'll just DM them to Kazkin.

As for the disks, eh, if we're gonna make them 1:1, I'd really RATHER not figure out on what gun to give to the Shield disk that would take a exceptionally large amount of points like the Nordwind does on the Marshal disk. Futhermore, Marshals do get a abundant more amount of disks than Shields do, but can make them 1:1 if you really want to Trilby.

Reply:
The pictures work, but you posted them in the wrong place.